sup3r/preprocessing/data_handling/dual_data_handling.py

```diff
@@ -55,6 +56,19 @@ def __init__(self,
             Spatial enhancement factor
         t_enhance : int
             Temporal enhancement factor
+        bc_files : list | tuple | str | None
+            One or more filepaths to .h5 files output by
+            MonthlyLinearCorrection or LinearCorrection. Used to bias correct
+            low resolution data prior to regrdding. These should contain
+            datasets named "{feature}_scalar" and "{feature}_adder" where
+            {feature} is one of the features contained by this DataHandler and
+            the data is a 3D array of shape (lat, lon, time) where time is
+            length 1 for annual correction or 12 for monthly correction. Bias
+            correction is only run if bc_files is not None.
```

Comment:
The code that makes the bias correction factors will operate on the original ERA domain though, won't it? Bias correction is really fast. Strong preference for keeping bc functions only in the base handler.

Reply:
Yes it will but the original ERA domain is not the same as the final domain after regridding to the wtk grid. I load the entire ERA domain in the lr_handler before regridding because its hard to select an ERA subdomain that still covers the wtk domain (due to the curvilinear grid). But if theres no concern about speed I'm fine with removing this.